pkg/config/sync.go

```diff
@@ -22,13 +22,15 @@ import (
 	"sync"
 
 	"github.com/devfile/devworkspace-operator/pkg/config/proxy"
+	"github.com/devfile/devworkspace-operator/pkg/constants"
 	routeV1 "github.com/openshift/api/route/v1"
 	k8sErrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	ctrl "sigs.k8s.io/controller-runtime"
 	crclient "sigs.k8s.io/controller-runtime/pkg/client"
 
+	"github.com/devfile/devworkspace-operator/apis/controller/v1alpha1"
 	controller "github.com/devfile/devworkspace-operator/apis/controller/v1alpha1"
```

Comment:
This is a duplicate import, you should use `&controller.StorageSizes{}`, etc. (instead of e.g. `&v1alpha1.StorageSizes{}`) in the code below.